crowbar_framework/app/controllers/machines_controller.rb

```diff
@@ -88,10 +88,19 @@ def role
 
   add_help(:rename, [:id])
   def rename
-    @machine.alias = params[:alias]
+    begin
+      @machine.alias = params[:alias]
+    rescue StandardError => e
```

Comment:
can we catch a specific exception here? or can it always be a different exceprtion?